scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -1,3 +1,19 @@
+#  Run Mutect 2 on a single tumor-normal pair
+#
+#  Description of inputs
+#  gatk4_jar: java jar file containing gatk 4 (protected)
```

Comment:
You may need GATK jar and picard jar (if applicable) to be String inputs.  If you put them as files, then you cannot run them through a docker image on local backends in cromwell.